doc/source/examples/multiphysics/rayleigh-plateau-instability/rayleigh-plateau-instability.rst

```diff
@@ -0,0 +1,333 @@
+
+In this example the Rayleigh-Plateau instability is simulated through a continuous glycerol jet that undergoes a perturbation of different excitation amplitudes. The velocity imposed at the jet inlet takes the following form:
+
+.. math::
+  u_\mathrm{inlet} = U \left[1+\delta_0 \sin{\left(2 \mathrm{\pi} f t \right)}\right] = U \left[1+\delta_0 \sin{\left(\frac{\kappa U t}{R_\mathrm{inlet}}\right)}\right]
+
+where :math:`U` is the initial uniform velocity of the jet, :math:`\delta_0` is the excitation amplitude, :math:`f = \frac{\kappa U}{2 \mathrm{\pi} R_\mathrm{inlet}}` is the excitation frequency with :math:`\kappa = \frac{2 \mathrm{\pi} R_\mathrm{inlet}}{\lambda}` the dimensionless wavenumber (:math:`\lambda` is the wavelength) and :math:`R_\mathrm{inlet}=1.145 \times 10^{-3} \; \mathrm m` the inlet radius. Lastly, :math:`t` is the time.
+
+.. note::
+  It is assumed that at the initial state the air is static :math:`\left(\mathbf{u}_\mathrm{air} = \mathbf{0} \; \mathrm{m\, s^{-1}}\right)`.
+
+The following figure displays the initial state:
+
+
++-------------------------------------------------------------------------------------------------------------------+
+|  .. figure:: images/initial-state-without-gravity.svg                                                             |
+|     :align: center                                                                                                |
+|     :width: 1000                                                                                                  |
+|     :name: Initial state of the jet                                                                               |
+|                                                                                                                   |
++-------------------------------------------------------------------------------------------------------------------+
+
+.. note::
+  In this example, gravity contributions are not considered :math:`\left(Fr = \frac{u}{\sqrt{g R_\mathrm{inlet}}} \rightarrow \infty \right)`.
+
+****
+
+--------------
+Parameter File
+--------------
+The parameter file for the case of :math:`\delta_0 = 0.20` is shown below.
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~
+The time integration is handled by a 2nd-order backward differentiation scheme (bdf2) with a maximum time-step of :math:`\Delta t = 4.4 \times 10^{-5} \; \text{s} \approx \Delta t_\sigma` which corresponds to the capillary time-step constraint (see :doc:`capillary wave example <../capillary-wave/capillary-wave>`).
+
+.. code-block:: text
+
+    subsection simulation control
+      set method           = bdf2
+      set time end         = 0.08
+      set time step        = 4.4e-5
+      set adapt            = true
+      set max cfl          = 0.75
+      set max time step    = 4.4e-5
+      set output name      = rayleigh-plateau
+      set output frequency = 5
+      set output path      = ./output_delta0_20/
```

Comment:
Is it necessary to have a different output folder name?

Reply:
Otherwise the files would overwrite themselves since many cases are run using the bash script, unless I change the output name for each case, but I think it is more clean to have separate folders instead of dumping them all in one. (This one was also generated using the bash script :))